lib/elixir/lib/option_parser.ex

```diff
@@ -173,18 +173,18 @@ defmodule OptionParser do
       {[debug: true], ["path/to/file"]}
 
       iex> OptionParser.parse!(["--limit", "xyz"], strict: [limit: :integer])
-      ** (OptionParser.ParseError) 1 error found!
-      --limit : Expected type integer, got "xyz"
+      ** (OptionParser.ParseError) found errors:
+      --limit - expected a value of type integer, got: "xyz"
```

Comment:
Elixir command line returns `--limit : Expected` so we should stick to that format or change it there. I think the `--limit - ` is confusing due to the multiple `-` in sequence